manifest.json

```
@@ -0,0 +1,15 @@
+{
+  "addons": [
+    {
+      "id": "todotreeview",
+      "version": "1.0",
```

```suggestion
      "version": "0.1.1",
```
To continue from the version in the plugins repo.